src/scss/components/_controls.scss

```diff
@@ -7,7 +7,7 @@
     position: absolute;
     top: 0;
     left: 0;
-    z-index: 2;
+    z-index: 3;
```

Comment:
_Three, it's the magic number..._

https://www.youtube.com/watch?v=daWObuUptrQ